models/issue.go

```diff
@@ -1498,3 +1512,45 @@ func updateIssue(e Engine, issue *Issue) error {
 func UpdateIssue(issue *Issue) error {
 	return updateIssue(x, issue)
 }
+
+// Get Blocked By Dependencies, aka all issues this issue is blocked by.
+func (issue *Issue) getBlockedByDependencies(e Engine) (_ []*Issue, err error) {
```

Comment:
IMO declaring variable names in function "header" (e.g. `(_ []*Issue, err error)`) is error prone and code does not look good (sometimes you type only `return`, sometimes you type `return xyz`). In this example you do not even use it and only one of two variables is declared.

Reply:
Fixed that.